src/components/forms/Fields/ExpandingSelectField.js

```diff
@@ -109,6 +113,11 @@ ExpandingSelectField.propTypes = {
     PropTypes.element,
     PropTypes.shape({ type: PropTypes.oneOf([FormattedMessage]) })
   ]).isRequired,
+  noItems: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.element,
+    PropTypes.shape({ type: PropTypes.oneOf([FormattedMessage]) })
```

Comment:
`oneOf` one element array?

Reply:
Actually, I just copy-pasted this shape. I am not sure, if it can be simplified (and I did not take the time to explore it since it is marginal).